build/repo.targets

```diff
@@ -1,6 +1,8 @@
-<Project>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is going to cause MSBuild warnings about double imports of Microsoft.Common.targets. We shouldn't be using Microsoft.NET.Sdk from repo.targets.